lib/teiserver/lobby/libs/lobby_restrictions.ex

```diff
@@ -193,23 +193,54 @@ defmodule Teiserver.Lobby.LobbyRestrictions do
         {:error,
          "* You cannot declare a lobby to be all welcome if there are player restrictions"}
 
-      is_noob_title?(name) ->
-        {:ok, get_noob_looby_tips()}
-
       true ->
-        {:ok, nil}
+        {:ok, get_tips(name)}
+    end
+  end
+
+  defp get_tips(name) do
+    tips =
+      case is_noob_title?(name) do
+        true -> get_noob_looby_tips()
+        false -> []
+      end
+
+    tips =
+      case is_rotato_title?(name) do
+        true -> tips ++ get_rotato_tips()
+        false -> tips
+      end
```

Comment:
I find these repeated shadowing and case a bit awkward and easy to mess up.
I think a nicer approach would be something like:

` tips = [] ++ noob_tips(name) ++ rotato_tips(name)` with these functions doing the check themselves and returning an empty list of that doesn't match.

```elixir
def rotato_tips(name) do
  if Regex.match?(~r/rotat/i, name) do
    # ....
  else
    []
  end
end
```